lms/djangoapps/instructor/views/api.py

```diff
@@ -2584,6 +2585,57 @@ def spoc_gradebook(request, course_id):
 
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+def get_blank_lti(request, course_id):  # pylint: disable=W0613
```

Comment:
Preferable to use human-readable version for pylint disables
`# pylint: disable=unused-argument`